vadovas/katalogas.rst

```diff
@@ -773,6 +773,30 @@ IRS rinkinių sąraše galima peržiūrėti tokią bendrą informaciją:
 *Sąrašas pildomas automatiškai, importuojant rinkinius iš atitinkamų portalų.*
 
 
+Duomenų išteklio rūšies forma
+**********************
+
+1. Organizacijos lange pasirinkite meniu punktą **[Duomenų rinkiniai]**.
+
+2. Duomenų išteklių sąrašo lange spauskite **[+ Pridėti duomenų išteklį]**.
```

Comment:
```suggestion
2. Duomenų išteklių sąrašo lange spauskite **[+ Pridėti duomenų išteklių]**.
```

Following this comment:
- https://github.com/atviriduomenys/katalogas/issues/1594#issuecomment-3156409191